corehq/apps/user_importer/importer.py

```diff
@@ -424,7 +424,11 @@ def _get_domain_info(domain):
                     user.add_phone_number(_fmt_phone(phone_number), default=True)
                 if name:
                     user.set_full_name(str(name))
-                data = user.metadata
+
+                # Add in existing data. Don't use metadata - we don't want to add profile-controlled fields.
+                for key, value in user.user_data.items():
```

Comment:
what happens if a field from the profile is included in the upload? Or is that handled in the `overlap` part of `update_metadata`

Reply:
And if it is handled there what does the uploader see?

Reply:
The overlap part handles it. It gets treated like other import errors: that row fails and the user sees an error saying there was a conflict and naming the property that conflicted.